Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -255,9 +255,10 @@ class BrowserViewController: UIViewController {
             DispatchQueue.main.asyncAfter(deadline: .now() + 3) {
                 if Preferences.Rewards.myFirstAdShown.value { return }
                 Preferences.Rewards.myFirstAdShown.value = true
-                AdsViewController.displayFirstAd(on: self) { [weak self] url in
-                    let request = URLRequest(url: url)
-                    self?.tabManager.addTabAndSelect(request, isPrivate: PrivateBrowsingManager.shared.isPrivateBrowsing)
```

Comment:
This change is on purpose, coming from #1597, looks like was an accident on rebase?

Reply:
It's on purpose. That other code is missing an argument for `displayFirstAd`

Reply:
Yeah sorry I mean the part where it uses `addTabAndSelect` instead of `openNewTab`